policyengine_canada/parameters/gov/cra/CPP/min_income.yaml

```diff
@@ -0,0 +1,9 @@
+description: Canada Pension Plan yearly minimum amount of pensionable earnings
+values:
+  2022-01-01: 3_500
```

Comment:
why is this needed, given it's already going to be zero if below this threshold due to the amount scale parameter structure?